fedimint-client-legacy/src/ln/mod.rs

```diff
@@ -550,7 +553,8 @@ mod tests {
         assert_eq!(contract_acc.contract.gateway_key, gateway.mint_pub_key);
         // TODO: test that the client has its key
 
-        let expected_amount_msat = invoice_amt_msat + (invoice_amt_msat / 100);
+        let expected_amount_msat = invoice_amt_msat
+            + (invoice_amt_msat * (gateway.fees_proportional_millionths / 1000000) as u64);
```

Comment:
> Can we use DEFAULT_FEES_PROPORTIONAL_MILLIONTHS constant here?

This value is actually injected (announced) by API call to register the gateway with federations. Client code should not have knowledge of the constant value.

> Ah I see its not the same value. Let's make a new constant then

if you mean we should make the computed value of `expected_amount_msat`, I don't see the sense since clients should expect the result of the computation to be dynamic. Updated with every fresh selection / update of the gateway to use in contracts

Reply:
Oh. That makes sense. No, it's a const

Reply:
See `compute_outgoing_contract_amount`